<?xml version="1.0"?>
<launch>

    <arg name="sim" default="false"/>
    <arg name="minibot" default="false"/>
    <arg name="wolfgang" default="true"/>

    <param name="/simulation_active" value="$(arg sim)" />
    
    <group if="$(arg wolfgang)">
        <rosparam file="$(find bitbots_hcm)/config/hcm_wolfgang.yaml" command="load"/>
        <node name="hcm" pkg="bitbots_hcm" type="hardware_control_manager.py" args="" output="screen" respawn="true">
        </node>
    </group>

    <node name="pause" pkg="bitbots_hcm" type="pause.py" respawn="true"/>

</launch>